src/madetomeasure_api/routes/subscribers.clj

```diff
@@ -29,8 +29,8 @@
 (defn create-subscribers [request]
   (let [subscribers (extract-subscribers request)]
     (try
```

Comment:
Yea I'll do that when I make another try catch. Probably extract it out into... the present logic I think